modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/PassThroughHttpSender.java

```diff
@@ -384,7 +385,8 @@ private void sendRequestContent(final MessageContext msgContext) throws AxisFaul
                 } else {
                     //if HTTP MEHOD = GET we need to write down the HEADER information to the wire and need
                     //to ignore any entity enclosed methods available.
-                    if (("GET").equals(msgContext.getProperty(Constants.Configuration.HTTP_METHOD)) || ("DELETE").equals(msgContext.getProperty(Constants.Configuration.HTTP_METHOD))) {
+                    if (("GET").equals(msgContext.getProperty(Constants.Configuration.HTTP_METHOD)) ||
```

Comment:
Fixed in https://github.com/wso2/wso2-synapse/pull/801/commits/b6c89cf2bdf920a6e2a44ab6b3f1fbc2fabcf145